### ISSUE TITLE
Fusion CLI to remove a subgraph from the package

### ISSUE THREAD

**Issue by AlexeyRaga** [ChilliCream/graphql-platform#6557]
### Product

Hot Chocolate

### Is your feature request related to a problem?

`fusion compose` provides a very easy way for composing a subgraph into a package. Doing it in CI is extremely helpful and useful!

However, there is no way to _remove_ a subgraph from the package.

In a distributed system with multiple subgraphs, when one service gets deprecated, it should be possible to remove its subgraph from the final GraphQL schema.

The only workaround that I have found is to somehow figure out what subgraphs and of what versions contribute to the final package and re-compose all of them from scratch (minus the obsolete one).
But it feels uneasy, error-prone and laborious, for the same reason as why `fusion` allows to compose one subgraph into an existing package instead of always requiring the full list of subgraphs for every composition,

### The solution you'd like

something like:

```
dotnet fusion remove -p <path-to-package> -s <name-of-the-subgraph-to-remove>
```

would be super handy.

**Comment by interad-woergoetter** [comment]
+1
I ran into the same issue just last week.
My solution was to create an empty schema for the subgraph I want to remove and pack that into the gateway package.

**Comment by glen-84** [comment]
Was this resolved by #6584, or is that different?

**Comment by michaelstaib** [comment]
Yes it was
